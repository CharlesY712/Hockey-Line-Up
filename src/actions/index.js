export const selectSeason = (season) => ({
  type: 'SELECT_SEASON',
  season
});