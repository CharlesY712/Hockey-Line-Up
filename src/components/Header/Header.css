

header {
  background-color: #2D3843;
  
}

.app-title {
  color: #A3A8AE;
  font-family: 'Allerta Stencil', sans-serif;
  font-size: 2rem;
  text-decoration: none;
}
.select-boxes {
  display: flex;
  justify-content: space-between;
}

.season-type {

}

.season-date {

}

/*
#1A1E21
#2D3843
#4E5357
#677584
#A3A8AE

rgb(31,27,27) nhl logo
rgb(228,229,229) nhl logo
rgb(210,210,210) nhl logo */