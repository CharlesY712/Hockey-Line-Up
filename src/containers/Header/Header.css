header {
  position: fixed;
  left: 0;
  top: 0;
  background-color: rgb(45, 45, 45);
}

.app-title {
  width: 100vw;
  display: flex;
  justify-content: center;
  color: #A3A8AE;
  font-family: 'Allerta Stencil', sans-serif;
  font-size: 2rem;
  text-decoration: none;
  padding-top: .5rem;
}

.select-boxes {
  width: 100vw;
  display: flex;
  justify-content: center;
}

.date-selector {
  color: #A3A8AE;
  background-color: rgb(45, 45, 45);
  margin-bottom: .5rem;
  margin-top: -.5rem;
}

@media (min-width: 414px) {
  .app-title {
    font-size: 3rem;
  }
  
  .date-selector {
    font-size: 2rem;
    margin-bottom: .5rem;
    margin-top: -.5rem;
  }
}

@media (min-width: 591px) {
}

@media (min-width: 768px) {
  .app-title {
    font-size: 3rem;
  }
  
  .date-selector {
    font-size: 2rem;
    margin-bottom: .5rem;
    margin-top: -.5rem;
  }
}
