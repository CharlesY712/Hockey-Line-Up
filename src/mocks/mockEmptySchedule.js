export const emptySchedule = {
  "dailygameschedule": {
    "lastUpdatedOn": "2018-04-15 11:54:02 PM"
  }
};