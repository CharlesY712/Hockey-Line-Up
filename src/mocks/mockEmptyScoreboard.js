export const emptyScoreboard = {
  "scoreboard": {
    "lastUpdatedOn": "2018-04-12 1:56:15 PM"
  }
};