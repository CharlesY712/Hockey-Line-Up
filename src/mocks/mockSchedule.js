export const schedule = {
  "dailygameschedule": {
    "lastUpdatedOn": "2018-04-12 4:24:49 PM",
    "gameentry": [
      {
        "id": "45866",
        "scheduleStatus": "Normal",
        "originalDate": null,
        "originalTime": null,
        "delayedOrPostponedReason": null,
        "date": "2018-04-14",
        "time": "3:00PM",
        "awayTeam": {
          "ID": "22",
          "City": "Colorado",
          "Name": "Avalanche",
          "Abbreviation": "COL"
        },
        "homeTeam": {
          "ID": "18",
          "City": "Nashville",
          "Name": "Predators",
          "Abbreviation": "NSH"
        },
        "location": "Bridgestone Arena"
      },
      {
        "id": "45875",
        "scheduleStatus": "Normal",
        "originalDate": null,
        "originalTime": null,
        "delayedOrPostponedReason": null,
        "date": "2018-04-14",
        "time": "3:00PM",
        "awayTeam": {
          "ID": "7",
          "City": "New Jersey",
          "Name": "Devils",
          "Abbreviation": "NJD"
        },
        "homeTeam": {
          "ID": "1",
          "City": "Tampa Bay",
          "Name": "Lightning",
          "Abbreviation": "TBL"
        },
        "location": "Amalie Arena"
      },
      {
        "id": "45874",
        "scheduleStatus": "Normal",
        "originalDate": null,
        "originalTime": null,
        "delayedOrPostponedReason": null,
        "date": "2018-04-14",
        "time": "8:00PM",
        "awayTeam": {
          "ID": "12",
          "City": "Toronto",
          "Name": "Maple Leafs",
          "Abbreviation": "TOR"
        },
        "homeTeam": {
          "ID": "11",
          "City": "Boston",
          "Name": "Bruins",
          "Abbreviation": "BOS"
        },
        "location": "TD Garden"
      },
      {
        "id": "45873",
        "scheduleStatus": "Normal",
        "originalDate": null,
        "originalTime": null,
        "delayedOrPostponedReason": null,
        "date": "2018-04-14",
        "time": "10:30PM",
        "awayTeam": {
          "ID": "26",
          "City": "San Jose",
          "Name": "Sharks",
          "Abbreviation": "SJS"
        },
        "homeTeam": {
          "ID": "29",
          "City": "Anaheim",
          "Name": "Ducks",
          "Abbreviation": "ANA"
        },
        "location": "Honda Center"
      }
    ]
  }
};