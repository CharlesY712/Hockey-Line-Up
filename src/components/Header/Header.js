import React, { Component } from 'react';
import { withRouter, Link } from 'react-router-dom';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import * as actions from '../../actions';
import './Header.css';
import { fetchSeason, fetchScoreboard } from '../../helpers/apiCalls';

class Header extends Component {

  async componentDidMount() {
    const fullSeason = await fetchSeason();
    const seasonGames = fullSeason.fullgameschedule.gameentry;
    this.props.addSeason(seasonGames);
  }
  
  determineSelectBox() {
    if (this.props.location.pathname === '/day') {
      return (<input 
        type="date" 
        name="day" 
        className="dateSelector" 
        id="date"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else if (this.props.location.pathname === '/week') {
      return (<input 
        type="week" 
        name="week" 
        className="dateSelector" 
        id="week"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else if (this.props.location.pathname === '/month') {
      return (<input 
        type="month" 
        name="month"  
        className="dateSelector" 
        id="month"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else {
      return null;
    }
  }

  render() {
    return (
      <header className="header">
        <Link to="/" className="app-title">HOCKEY LINE-UP</Link><br/>
        <div className="select-boxes">
          {this.determineSelectBox()}
          <div>Season: 
            <select 
              className="season-date"
              onChange={event => this.props.setSeasonYear(event.target.value)}>
              <option value="2017-2018">2017-2018</option>
              <option value="2016-2017">2016-2017</option>
              <option value="2015-2016">2015-2016</option>
              <option value="2014-2015">2014-2015</option>
              <option value="2013-2014">2013-2014</option>
              <option value="2012-2013">2012-2013</option>
              <option value="2011-2012">2011-2012</option>
              <option value="2010-2011">2010-2011</option>
              <option value="2009-2010">2009-2010</option>
              <option value="2008-2009">2008-2009</option>
              <option value="2007-2008">2007-2008</option>
            </select>
            <select
              className="season-type"
              onChange={event => this.props.setSeasonType(event.target.value)}>
              <option value="-playoff">Playoff</option>
              <option value="-regular">Regular</option>
            </select>
          </div>
        </div>
      </header>
    );
  }
}

Header.propTypes = {
  addSeason: PropTypes.func,
  setDate: PropTypes.func,
  setSeasonYear: PropTypes.func,
  setSeasonType: PropTypes.func,
  location: PropTypes.object
};

export const mapDispatchToProps = dispatch => ({
  addSeason: (season) => dispatch(actions.addSeason(season)),
  setDate: (date) => dispatch(actions.setDate(date)),
  setSeasonYear: (year) => dispatch(actions.setSeasonYear(year)),
  setSeasonType: (type) => dispatch(actions.setSeasonType(type))
});

export default withRouter(connect(null, mapDispatchToProps)(Header));
