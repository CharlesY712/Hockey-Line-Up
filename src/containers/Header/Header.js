import React, { Component } from 'react';
import { withRouter, Link } from 'react-router-dom';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import * as actions from '../../actions';
import './Header.css';

class Header extends Component {

  determineSelectBox() {
    if (this.props.location.pathname === '/day') {
      return (<input 
        type="date" 
        name="day" 
        className="dateSelector" 
        id="date"
        min="2017-10-04"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else if (this.props.location.pathname === '/week') {
      return (<input 
        type="week" 
        name="week" 
        className="dateSelector" 
        id="week"
        min="2017-10-04"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else if (this.props.location.pathname === '/month') {
      return (<input 
        type="month" 
        name="month"  
        className="dateSelector" 
        id="month"
        min="2017-10"
        onChange={event => this.props.setDate(event.target.value)}
      />);
    } else {
      return null;
    }
  }

  render() {
    return (
      <header className="header">
        <Link to="/" className="app-title">HOCKEY LINE-UP</Link><br/>
        <div className="select-boxes">
          {this.determineSelectBox()}
        </div>
      </header>
    );
  }
}

Header.propTypes = {
  setDate: PropTypes.func,
  location: PropTypes.object
};

export const mapDispatchToProps = dispatch => ({
  setDate: (date) => dispatch(actions.setDate(date))
});

export default withRouter(connect(null, mapDispatchToProps)(Header));
