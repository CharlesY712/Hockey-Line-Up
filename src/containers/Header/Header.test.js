import { mapDispatchToProps } from './Header';
import { schedule }  from '../../mocks/mockSchedule';
import * as actions from '../../actions';

describe('Header', () => {

  describe('Header container', () => {

  });

  describe('mapDispatchToProps', () => {

    it('should call dispatch with the correct params on setDate', () => {
      const mockDispatch = jest.fn();
      const mapped = mapDispatchToProps(mockDispatch);
      const expected = actions.setDate(schedule);

      mapped.setDate(schedule);

      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });
  });
});