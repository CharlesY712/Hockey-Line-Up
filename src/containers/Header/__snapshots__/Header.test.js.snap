// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header container should match the snapshot for day 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    location={
        Object {
            "pathname": "/day",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="app-title"
          replace={false}
          to="/"
>
          HOCKEY LINE-UP
</Link>,
        <br />,
        <div
          className="select-boxes"
>
          <input
                    className="date-selector"
                    id="date"
                    min="2017-10-04"
                    name="day"
                    onChange={[Function]}
                    type="date"
          />
</div>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "HOCKEY LINE-UP",
          "className": "app-title",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "HOCKEY LINE-UP",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <input
            className="date-selector"
            id="date"
            min="2017-10-04"
            name="day"
            onChange={[Function]}
            type="date"
/>,
          "className": "select-boxes",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "date-selector",
            "id": "date",
            "min": "2017-10-04",
            "name": "day",
            "onChange": [Function],
            "type": "date",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="app-title"
            replace={false}
            to="/"
>
            HOCKEY LINE-UP
</Link>,
          <br />,
          <div
            className="select-boxes"
>
            <input
                        className="date-selector"
                        id="date"
                        min="2017-10-04"
                        name="day"
                        onChange={[Function]}
                        type="date"
            />
</div>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "HOCKEY LINE-UP",
            "className": "app-title",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "HOCKEY LINE-UP",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <input
              className="date-selector"
              id="date"
              min="2017-10-04"
              name="day"
              onChange={[Function]}
              type="date"
/>,
            "className": "select-boxes",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "date-selector",
              "id": "date",
              "min": "2017-10-04",
              "name": "day",
              "onChange": [Function],
              "type": "date",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Header Header container should match the snapshot for home 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    location={
        Object {
            "pathname": "/",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="app-title"
          replace={false}
          to="/"
>
          HOCKEY LINE-UP
</Link>,
        <br />,
        <div
          className="select-boxes"
/>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "HOCKEY LINE-UP",
          "className": "app-title",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "HOCKEY LINE-UP",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": null,
          "className": "select-boxes",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="app-title"
            replace={false}
            to="/"
>
            HOCKEY LINE-UP
</Link>,
          <br />,
          <div
            className="select-boxes"
/>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "HOCKEY LINE-UP",
            "className": "app-title",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "HOCKEY LINE-UP",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": null,
            "className": "select-boxes",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Header Header container should match the snapshot for month 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    location={
        Object {
            "pathname": "/month",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="app-title"
          replace={false}
          to="/"
>
          HOCKEY LINE-UP
</Link>,
        <br />,
        <div
          className="select-boxes"
>
          <input
                    className="date-selector"
                    id="month"
                    min="2017-10"
                    name="month"
                    onChange={[Function]}
                    type="month"
          />
</div>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "HOCKEY LINE-UP",
          "className": "app-title",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "HOCKEY LINE-UP",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <input
            className="date-selector"
            id="month"
            min="2017-10"
            name="month"
            onChange={[Function]}
            type="month"
/>,
          "className": "select-boxes",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "date-selector",
            "id": "month",
            "min": "2017-10",
            "name": "month",
            "onChange": [Function],
            "type": "month",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="app-title"
            replace={false}
            to="/"
>
            HOCKEY LINE-UP
</Link>,
          <br />,
          <div
            className="select-boxes"
>
            <input
                        className="date-selector"
                        id="month"
                        min="2017-10"
                        name="month"
                        onChange={[Function]}
                        type="month"
            />
</div>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "HOCKEY LINE-UP",
            "className": "app-title",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "HOCKEY LINE-UP",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <input
              className="date-selector"
              id="month"
              min="2017-10"
              name="month"
              onChange={[Function]}
              type="month"
/>,
            "className": "select-boxes",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "date-selector",
              "id": "month",
              "min": "2017-10",
              "name": "month",
              "onChange": [Function],
              "type": "month",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Header Header container should match the snapshot for week 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    location={
        Object {
            "pathname": "/week",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="app-title"
          replace={false}
          to="/"
>
          HOCKEY LINE-UP
</Link>,
        <br />,
        <div
          className="select-boxes"
>
          <input
                    className="date-selector"
                    id="week"
                    min="2017-10-04"
                    name="week"
                    onChange={[Function]}
                    type="week"
          />
</div>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "HOCKEY LINE-UP",
          "className": "app-title",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "HOCKEY LINE-UP",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <input
            className="date-selector"
            id="week"
            min="2017-10-04"
            name="week"
            onChange={[Function]}
            type="week"
/>,
          "className": "select-boxes",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "date-selector",
            "id": "week",
            "min": "2017-10-04",
            "name": "week",
            "onChange": [Function],
            "type": "week",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="app-title"
            replace={false}
            to="/"
>
            HOCKEY LINE-UP
</Link>,
          <br />,
          <div
            className="select-boxes"
>
            <input
                        className="date-selector"
                        id="week"
                        min="2017-10-04"
                        name="week"
                        onChange={[Function]}
                        type="week"
            />
</div>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "HOCKEY LINE-UP",
            "className": "app-title",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "HOCKEY LINE-UP",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <input
              className="date-selector"
              id="week"
              min="2017-10-04"
              name="week"
              onChange={[Function]}
              type="week"
/>,
            "className": "select-boxes",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "date-selector",
              "id": "week",
              "min": "2017-10-04",
              "name": "week",
              "onChange": [Function],
              "type": "week",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
